runtime/include/vendor_version.h

```diff
@@ -49,17 +49,14 @@
 #define VENDOR_SHORT_NAME "OpenJ9"
 
 #define JAVA_VM_VENDOR "Eclipse OpenJ9"
+#define JAVA_VM_NAME "Eclipse OpenJ9 VM"
 
-#if JAVA_SPEC_VERSION < 12
```

Comment:
What is the purpose of this change?

Reply:
`JAVA_VENDOR` & `JAVA_VENDOR_URL` was decorated by `#if JAVA_SPEC_VERSION < 12` in `vmprops.c`, now it is a runtime check instead hence these two need to be defined for `JAVA_SPEC_VERSION  12+` cases as well.